CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.5.1](https://github.com/fiatconnect/fiatconnect-sdk/compare/v0.5.0...v0.5.1) (2023-01-04)
+
+
+### Bug Fixes
```

Comment:
these seem to be incorrectly labeled as bug fixes

Reply:
It seems like a Patch version changes default to Bug Fixes everywhere else in `CHANGELOG.md`, but I can change the title to something else